dist-karaf/apache-brooklyn/src/main/resources/etc/default.catalog.bom

```diff
@@ -0,0 +1,371 @@
+
+# this catalog bom is an illustration supplying a few useful sample items
```

Comment:
Would be great if we could re-use the same `default.catalog.bom` when building classic and karaf distributions.